Tools/gdb/libpython.py

```diff
@@ -709,13 +709,18 @@ def write_repr(self, out, visited):
             pyop_value.write_repr(out, visited)
         out.write('}')
 
+    try:
+        _gdb_error = gdb.error
+    except AttributeError:
+        _gdb_error = RuntimeError
```

Comment:
OK.
There are many `except RuntimeError` in this file and most of them may be `gdb.error`.
So let's make `_gdb_error` global instead of class attribute for it can be used from other places in the future.